### DISCUSSION TITLE
405 on dmn quickstart

### DISCUSSION THREAD

**Discussion by magick93** [quarkusio/quarkus#22679] [Community]
Hello

I'm running through the guide on https://quarkus.io/guides/kogito-dmn and I am unable to make the test pass. It fails with:

```
[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   PricingTest.testBasePrice:22 1 expectation failed.
Expected status code <200> but was <405>.

```

I'm using the following Java version - `17.0.1-tem`

**Comment by gsmet** [comment]
Yeah, unfortunately, Kogito support is broken in 2.6. We will release 2.6.2 soon with a fix.

In the meantime, please create your projects with 2.5.4.Final.

**Comment by magick93** [comment]
Thanks. I tried with 2.5.4.Final and still get the same result.

**Reply by gsmet** [reply]
OK, thanks for checking. I was unable to upgrade the quickstarts of Kogito for a while due to various issues. Hopefully, once the new version is released, I will be able to and I'll get someone from the Kogito team to have a look at this.
